rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidPatientRegisterActivityTest.java

```diff
@@ -33,12 +34,18 @@
     @Before
     public void setUp() {
         MockitoAnnotations.initMocks(this);
+
         covidPatientRegisterActivity = Robolectric.buildActivity(CovidPatientRegisterActivity.class)
                 .create()
                 .resume()
                 .get();
     }
 
+    @Test
+    public void testUserAuthorizationVerificationTaskShouldReturnFinishedStatus() {
+        Assert.assertEquals(AsyncTask.Status.FINISHED, Utils.UserAuthorizationVerificationTask.getInstance(null).getStatus());
```

Comment:
This should just verify that `Utils.verifyAuthorization` is invoked.

Reply:
actually, when I combine powermock with this test class then the throws a class not prepared exception.

Reply:
Use shadows instead of powermock.

Reply:
fixed.